### PR TITLE
Accommodates multiple currencies

### PR DESCRIPTION
**Pull request by ashfurrow**
This makes the app respect the `currency` property on a sale artwork. This also removes some dead code I found, too.

Marked as a WIP because I want to test against CHF on the fulfillment page (we use a separate `UILabel` for the currency symbol and I don't know how autolayout will react to a wider symbol).

### CODE REVIEW

**Comment by ashfurrow** [comment]
Yeah, looks like a problem:

simulator screen shot jan 30 2017 6 25 46 am

It's only that one spot, I'd rather fix this all at once than leave it.

**Comment by orta** [comment]
cool beans

**Comment by ashfurrow** [comment]
Looking sharp:

simulator screen shot jan 30 2017 2 33 08 pm

**Comment by ashfurrow** [comment]
This is good for final review + merging, will deploy after I land back in NYC 👍

File: Kiosk/App/Models/SaleArtworkViewModel.swift [hunk 1]
```diff
@@ -22,13 +22,13 @@ extension SaleArtworkViewModel {
...
+            let dollars = centsToPresentableDollarsString(estimateCents, currencySymbol: saleArtwork.currencySymbol)
             return "Estimate: \(dollars)"
 
         // Try to extract non-nil low/high estimates.
         case (_, .some(let lowCents), .some(let highCents)):
-            let lowDollars = NumberFormatter.currencyString(forDollarCents: lowCents as NSNumber!)!
-            let highDollars = NumberFormatter.currencyString(forDollarCents: highCents as NSNumber!)!
+            let lowDollars = centsToPresentableDollarsString(lowCents, currencySymbol: saleArtwork.currencySymbol)
+            let highDollars = centsToPresentableDollarsString(highCents, currencySymbol: saleArtwork.currencySymbol)
             return "Estimate: \(lowDollars)–\(highDollars)"
```
Comment (orta) [inline Kiosk/App/Models/SaleArtworkViewModel.swift]: ah yeah, I remember doing all this with eigen 👍

File: Kiosk/Storyboards/Fulfillment.storyboard [hunk 2]
```diff
@@ -2140,9 +2141,9 @@
         <simulatedScreenMetrics key="destination" type="retina4_7.fullscreen"/>
     </simulatedMetricsContainer>
     <inferredMetricsTieBreakers>
-        <segue reference="eZu-3X-lXX"/>
         <segue reference="9X5-eM-AKF"/>
         <segue reference="hvW-mC-OYp"/>
         <segue reference="ccv-le-Cu0"/>
+        <segue reference="eZu-3X-lXX"/>
```
Comment (orta) [inline Kiosk/Storyboards/Fulfillment.storyboard]: guess at some point they decided to order these

File: Podfile.lock [hunk 3]
```diff
@@ -171,7 +171,7 @@ SPEC CHECKSUMS:
   NSObject+Rx: 2a9cd801d9c847e6d2486cbad8d7701b67834e70
   ORStackView: 5df6b1b990b0648d8ef6f69f89361ea8648e8f64
   Quick: 5d290df1c69d5ee2f0729956dcf0fd9a30447eaa
-  Reachability: 5b69def55ffa04d6f9425a6da1f47338b60557b2
+  Reachability: 2fc2badcac191cd1a15044bc7fe45aa96c499ece
```
Comment (orta) [inline Podfile.lock]: this one is interesting, I wonder what changed in the Podspec?

File: circle.yml [hunk 4]
```diff
@@ -9,13 +9,13 @@ dependencies:
...
-    - xcrun instruments -w "30244F9E-BA58-403B-82B1-ACD9AE948EB6" || echo "(Pre)Launched the simulator."
+    - xcrun instruments -w "93389280-6FDB-45D5-BF91-8890B50BEAC6" || echo "(Pre)Launched the simulator."
   post:
-    - set -o pipefail && xcodebuild -destination "id=30244F9E-BA58-403B-82B1-ACD9AE948EB6" -scheme "Kiosk" -workspace "Kiosk.xcworkspace" -derivedDataPath derived_data build | xcpretty --color
+    - set -o pipefail && xcodebuild -destination "id=93389280-6FDB-45D5-BF91-8890B50BEAC6" -scheme "Kiosk" -workspace "Kiosk.xcworkspace" -derivedDataPath derived_data build | xcpretty --color
 
 test:
   override:
-    - set -o pipefail && xcodebuild -destination "id=30244F9E-BA58-403B-82B1-ACD9AE948EB6" -scheme "Kiosk" -workspace "Kiosk.xcworkspace" -derivedDataPath derived_data test | bundle exec second_curtain | xcpretty --color --test
+    - set -o pipefail && xcodebuild -destination "id=93389280-6FDB-45D5-BF91-8890B50BEAC6" -scheme "Kiosk" -workspace "Kiosk.xcworkspace" -derivedDataPath derived_data test | bundle exec second_curtain | xcpretty --color --test
```
Comment (orta) [inline circle.yml]: hah, sure

File: Podfile.lock (hunk 3 above)
Comment (orta) [inline Podfile.lock]: lols, hasn't changed in 4 years - https://github.com/ashfurrow/Reachability/blob/frameworks/Reachability.podspec

Comment (orta) [inline Podfile.lock]: Considering tests pass, and there's not been a commit on the fork - I'm 👍 with this - but I'd recommend _you_ `rm -rf Pods` ash, http://artsy.github.io/blog/2016/05/03/podspec-checksums/

Comment (ashfurrow) [inline Podfile.lock]: Thanks Orta, good idea.

**Pull request merged**